src/current/_includes/releases/v24.1/v24.1.0-alpha.1.md

```diff
@@ -140,106 +138,57 @@ Release Date: March 7, 2024
-- This change separates the rangefeed traffic into its own RPC connection class. This improves isolation and reduces interference with the foreground SQL traffic, which reduces chances of head-of-line blocking caused by unrelated traffic. The new COCKROACH_RANGEFEED_USE_DEFAULT_CONNECTION_CLASS env variable can be set to use the DefaultClass instead (which was the previous default choice for rangefeeds). [#117730][#117730]
-- This change makes changefeed initial scan traffic, which can be bulky, go through a different RPC/TCP connection than the foreground SQL/KV traffic. This reduces interference between workloads, and reduces chances of head-of-line blocking issues. [#117810][#117810]
-- The optimizer now generates more efficient query plans for queries with comparison of timestamp and interval columns, e.g., `timestamp_col - '1 day'::INTERVAL > now()`. [#118307][#118307]
-
-<h3 id="v24-1-0-alpha-1-miscellaneous">Miscellaneous</h3>
-
-<h4 id="v24-1-0-alpha-1-breaking-change">Breaking change</h4>
-
-- The --config-profile command line flag added for preview support of PCR in the previous release has been removed. Nodes, including those for clusters on which PCR may be used, are instead started normally, with the one-time steps to configure the cluster for PCR being performed via SQL statements instead. Specifically, to use PCR, on the source cluster one must 1) create an application tenant, 2) switch it to be the default, and 3) enable rangefeeds. [#119366][#119366]
-
-<h4 id="v24-1-0-alpha-1-doc">Doc</h4>
-
-- Cluster setting for changefeed.balance_range_distribution.enable now has a clearer description. [#114599][#114599]
-
-<h4 id="v24-1-0-alpha-1-missing-category">Missing category</h4>
-
-- `kafka_sink_config` now supports specifying a different client ID for different changefeeds, enabling users to define distinct kafka quota configurations for various changefeeds.  For any kafka versions >= V1_0_0_0 ([KIP-190: Handle client-ids consistently between clients and brokers](https://cwiki.apache.org/confluence/display/KAFKA/KIP-190%3A+Handle+client-ids+consistently+between+clients+and+brokers)), any string can be used as client ID. For earlier kafka versions, clientID can only contain characters [A-Za-z0-9._-] are acceptable.  For example, ``` CREATE CHANGEFEED FOR ... WITH kafka_sink_config='{"ClientID": "clientID1"}' ``` [#118643][#118643]
-- Changefeed.kafka_throttling_hist_nanos has now been added to metrics to monitor sarama throttling behavior resulting from exceeding kafka quota. [#117693][#117693]
--  [#113993][#113993]
+- Sorting by [locality]({% link v23.2/table-localities) can improve the performance and predictability of [follower reads]({% link v23.2/follower-reads.md}). [#112993][#112993]
+- Allow stores to start in parallel reducing start times for nodes with many stores. [#115285][#115285]
```

Comment:
This is opaque and needs a rewrite. Assume it's something like "CockroachDB now allows stores..."